v3/plugins/redhat/quarkus/1.1.1/meta.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: v2
+publisher: redhat
+name: quarkus
+version: 1.1.1
+type: VS Code extension
+displayName: Quarkus Tools
+title: Quarkus Tools by Red Hat
+description: This plug-in provides support for Quarkus development via a Quarkus language server and a Java jdt.ls.
+icon: https://www.eclipse.org/che/images/logo-eclipseche.svg
+repository: https://github.com/redhat-developer/vscode-quarkus
+category: Language
+firstPublicationDate: "2019-11-05"
+spec:
+  containers:
+    - image: "docker.io/eclipse/che-remote-plugin-runner-java8:next"
+      name: vscode-quarkus
+      memoryLimit: "1500Mi"
+      volumes:
+      - mountPath: "/home/theia/.m2"
+        name: m2
+  extensions:
+    - https://github.com/microsoft/vscode-java-debug/releases/download/0.23.0/vscode-java-debug-0.23.0.vsix
+    - https://download.jboss.org/jbosstools/static/jdt.ls/stable/java-0.50.0-1825.vsix
```

Comment:
I don't think it makes sense to have a separate Quarkus extensions. We need to include this support in the Java extension. Otherwise, we'll have to have a separate plugin for all combinations of technologies. We should be mix-and-match